packages/playwright-core/src/server/chromium/crServiceWorker.ts

```diff
@@ -61,41 +58,28 @@ export class CRServiceWorker extends Worker {
     super.didClose();
   }
 
-  async updateOffline(initial: boolean): Promise<void> {
+  async updateOffline(): Promise<void> {
     if (!this._isNetworkInspectionEnabled())
       return;
-
-    const offline = !!this._browserContext._options.offline;
-    if (!initial || offline)
-      await this._networkManager?.setOffline(offline);
+    await this._networkManager?.setOffline(!!this._browserContext._options.offline).catch(() => {});
   }
 
-  async updateHttpCredentials(initial: boolean): Promise<void> {
+  async updateHttpCredentials(): Promise<void> {
     if (!this._isNetworkInspectionEnabled())
       return;
-
-    const credentials = this._browserContext._options.httpCredentials || null;
-    if (!initial || credentials)
-      await this._networkManager?.authenticate(credentials);
+    await this._networkManager?.authenticate(this._browserContext._options.httpCredentials || null).catch(() => {});
   }
 
-  async updateExtraHTTPHeaders(initial: boolean): Promise<void> {
+  async updateExtraHTTPHeaders(): Promise<void> {
     if (!this._isNetworkInspectionEnabled())
       return;
-
-    const headers = network.mergeHeaders([
-      this._browserContext._options.extraHTTPHeaders,
-      this._extraHTTPHeaders,
```

Comment:
why did this go away?

Reply:
This field was always `null`, so I removed it.